server/lib/oauth/model.ts

```diff
@@ -159,16 +159,22 @@ const model: OauthModel = {
       scope,
     });
 
-    await models.Activity.create({
-      type: activities.OAUTH_APPLICATION_AUTHORIZED,
-      UserId: user.id,
-      CollectiveId: user.CollectiveId,
-      data: {
-        application: application.publicInfo,
-        collective: collective.minimal,
-        scope,
-      },
+    // Only send the email if there is no active user token right now
+    const userToken = await models.UserToken.findOne({
+      where: { ApplicationId: application.id, UserId: user.id },
     });
```

Comment:
A `model.exists` would be nice for those cases (https://github.com/sequelize/sequelize/issues/10187)